OnboardIos/OnboardIos/Models/Users.h

```diff
@@ -9,7 +9,9 @@
 #import <Foundation/Foundation.h>
 
 @interface Users : NSObject
-
+{
+    NSMutableArray *userViewCount;
```

Comment:
when you create a variable { inside the interface } is it a good practice if you use underline before the name, so

NSMutableArray *_userViewCount;

or you can change this variable to be a Property... take a look at this article : http://williamboles.me/objective-c-coding-style/